docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -260,8 +271,125 @@ Server can set age rating for current playback. Client can access this rating (r
     }
     ```
 
+## Managing playlists on server side
+
+To manage the media controller playlists in your server application, you must learn to create, save, delete playlists.
```

Comment:
to create, save, **and** delete playlists.

Reply:
Done